elide-core/src/main/java/com/yahoo/elide/core/filter/Operator.java

```diff
@@ -316,6 +334,24 @@ public static Operator fromString(final String string) {
         return (T entity) -> false;
     }
 
+    private static <T> Predicate<T> isEmpty(String field, RequestScope requestScope) {
+        return (T entity) -> {
+            Object val = getFieldValue(entity, field, requestScope);
+            if (val == null) { return true; }
+            if (val instanceof String) {
```

Comment:
i feel like only collection and map should be handled here

Reply:
So the string `""` is not considered empty?

Reply:
So we generate JPQL queries based on IS EMPTY and IS NOT EMPTY.  I'm not sure those will work on Strings when we push this to the datastore.  

I'm inclined to think the JPQL queries will blow up if we try this on Strings in the DB.  Either remove the String case - or we need to add explicit tests to see that they work as expected.

Reply:
@Chandrasekar-Rajasekar Let's remove the string case.